TWLight/static/css/new-local.css

```
@@ -27,10 +33,161 @@ body {
   }
 }
 
+/*
+----------------------------------------------------------------------------
+LOGIN PARTIAL CSS
+--------------------------------------------------------------------------
+*/
```

A small bottom margin for the login section will keep the bottom of the login button from disappearing in mobile view, and doesn't have much of an impact  on the others. Could go in a media query or be approached differently.
```suggestion
*/
.login-section{
    margin-bottom: 5px;
}
```

---

Done!